application/src/main/java/org/togetherjava/tjbot/commands/moderation/modmail/ModMailCommand.java

```diff
@@ -116,11 +143,10 @@ public void onSlashCommand(SlashCommandInteractionEvent event) {
             return;
         }
 
-        MessageCreateAction message =
-                createModMessage(event, userId, modMailAuditLog.orElseThrow());
+        MessageCreateAction message = createModMessage(event, Boolean.parseBoolean(args.get(0)),
+                userId, modMailAuditLog.orElseThrow());
```

Comment:
The code is easier to read if you extract the args seperated and at the beginning:
```java
boolean stayAnonymous = Boolean.parseBoolean(args.get(0));
long guildToContact = Long.parseLong(args.get(1));
...
```